src/main/java/com/amazon/opendistroforelasticsearch/sql/antlr/semantic/types/function/ScalarFunction.java

```diff
@@ -33,13 +33,14 @@
     ABS(func(T(NUMBER)).to(T)), // translate to Java: <T extends Number> T ABS(T)
     ASIN(func(T(NUMBER)).to(T)),
     ATAN(func(T(NUMBER)).to(T)),
-    ATAN2(func(T(NUMBER)).to(T)),
+    ATAN2(func(T(NUMBER), T(NUMBER)).to(T)),
```

Comment:
The second parameter should be just NUMBER without generic type (T).

Reply:
@dai-chen Done. Thanks!